server/src/server.js

```diff
@@ -807,6 +807,8 @@ app.post("/api/set-title/:id/:domain", function(req, res) {
       return null;
     }
     shot.userTitle = userTitle;
+    // Here we remove potentially confidential data
+    shot.openGraph = {};
```

Comment:
I think the title should be updated.

If there are concerns regarding the data saved along with the shot, then please file another bug so it can be triaged.